src/mupdf_wrapper/build.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
+
+CC=${CC:-gcc}
+LD=${LD:-ld}
```

Comment:
No longer needed.

Reply:
Ok for LD.
CC is, however: `mupdf_wrapper/build-kobo.sh`, which calls on this `mupdf_wrapper/build.sh`, actually set CC and AR to the arm toolchain.

Reply:
My comment is on the *LD* line.